var/spack/repos/builtin/packages/fp16/package.py

```diff
@@ -0,0 +1,38 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Fp16(Package):
+    """FP16 is a header-only library for
+    conversion to/from half-precision floating point formats"""
+
+    homepage = "https://github.com/Maratyszcza/FP16/"
+    url      = "https://github.com/Maratyszcza/FP16.git"
+
+    version('master', git='https://github.com/Maratyszcza/FP16.git')
```

Comment:
isn't there a release available?

Reply:
No release found for fp16.

Reply:
You may want to ask them for one. I'm not sure what the policy is supposed to be in these cases, I'd merge as is, but maybe @adamjstewart or @tgamblin have different wishes